resources/authgear/templates/en/web/authflowv2/enter_password.html

```diff
@@ -32,15 +32,7 @@ <h1 class="screen-title">
       "HasErrorMessage" $.PasswordInputError.HasErrorMessage
     ) }}
     {{ if $.DeviceTokenEnabled }}
-      <div class="flex flex-row items-center gap-x-2.5 w-full">
-        <div class="checkbox">
-          <input id="device-token" type="checkbox" name="x_device_token" value="true">
-          <label class="checkbox__label" for="device-token">
-            <span class="checkbox__icon material-icons">check</span>
-          </label>
-        </div>
-        <label class="text-sm primary-txt" for="device-token">{{ template "create-device-token-label" }}</label>
-      </div>
+      {{ template "authflowv2/__device_token_checkbox.html" (dict "Classname" "w-full") }}
```

Comment:
w-full should be replaced by self-stretch, because we want to negate the effect of items-center.